ghga_connector/core/httpyexpect_translation.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2021 - 2022 Universität Tübingen, DKFZ and EMBL
```

Comment:
You mean instead of having specs wrapped into classes, just have the ResponseExceptionTranslator and create an instance passing both spec and response at the call site?